core/update_ffmeta_awsem/service.py

```diff
@@ -257,10 +269,12 @@ def real_handler(event, context):
     # get data
     # used to automatically determine the environment
     tibanna_settings = event.get('_tibanna', {})
-    tibanna = utils.Tibanna(**tibanna_settings)
-    # sbg = sbg_utils.create_sbg_workflow(token=tibanna.sbg_keys, **event.get('workflow'))
-    ff_meta = ff_utils.create_ffmeta_awsem(app_name=event.get('ff_meta').get('awsem_app_name'), **event.get('ff_meta'))
-    pf_meta = [ff_utils.ProcessedFileMetadata(**_) for _ in event.get('pf_meta')]
+    tibanna = utils.Tibanna(tibanna_settings['env'], settings=tibanna_settings)
+    ff_meta = tibanna_utils.create_ffmeta_awsem(
+        app_name=event.get('ff_meta').get('awsem_app_name'),
+        **event.get('ff_meta')
+    )
+    pf_meta = [tibanna_utils.ProcessedFileMetadata(**_) for _ in event.get('pf_meta')]
```

Comment:
little nitpickery here, but _ is meant to be a variable that you are not going to use... and it is used here, so probably change the name to item or something else...

Reply:
Agreed. I didn't create that var name but can definitely change it.